packages/tailwindcss-transformer/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@clerk/tailwindcss-transformer",
+  "version": "0.1.0",
+  "private": true,
+  "license": "MIT",
+  "author": {
+    "name": "Clerk, Inc.",
+    "email": "support@clerk.com",
+    "url": "git+https://github.com/clerk/javascript.git"
+  },
+  "scripts": {
+    "test": "vitest run --pool forks"
+  },
+  "dependencies": {
+    "@babel/parser": "^7.24.5",
+    "postcss": "^8.4.38",
+    "postcss-value-parser": "^4.2.0",
+    "recast": "^0.23.7",
+    "tailwindcss": "^3.4.3",
+    "vitest": "^1.6.0"
```

Comment:
We're using Jest everywhere else so Vitest needs to be changed here to Jest

Reply:
@LekoArts i checked in with @BRKalow about vitest and his conclusion was this wasn't a problem. its probably worth surfacing as a topic with a few more folks before we make any changes here @alexcarpenter (particularly bc this is all a bit exploratory atm)